templates/crossplane/pkg/controller.go.tpl

```diff
@@ -159,8 +171,14 @@ func newExternal(kube client.Client, client svcsdkapi.{{ .SDKAPIInterfaceTypeNam
 	e := &external{
 		kube:           kube,
 		client:         client,
+		{{- if or .CRD.Ops.ReadOne .CRD.Ops.GetAttributes .CRD.Ops.ReadMany }}
 		preObserve:     nopPreObserve,
 		postObserve:    nopPostObserve,
+		lateInitialize: nopLateInitialize,
+    isUpToDate:     alwaysUpToDate,
+    {{- else }}
+    observe:        nopObserve,
+    {{- end }}
```

Comment:
looks like indentation is goofy above?

Reply:
Went over all indentations and fixed them.